commands/pipelines/setup.js

```diff
@@ -0,0 +1,231 @@
+    type: 'confirm',
+    name: 'ci',
+    message: 'Enable automatic Heroku CI test runs?'
+  }])
+
+  if (settings.ci && organization) {
+    settings.organization = organization
+  }
+
+  return settings
+}
+
+function setupPipeline (token, app, settings, pipelineID, ciSettings = {}) {
+  const promises = [kolkrabbi.updateAppLink(token, app, settings)]
+
+  if (ciSettings.ci) {
+    promises.push(
+      kolkrabbi.updatePipelineRepository(token, pipelineID, ciSettings)
+    )
+  }
+
+  return Promise.all(promises).then(([appLink]) => {
+    return appLink
+  }, (error) => {
+    cli.error(error.response.body.message)
+  })
+}
+
+module.exports = {
+  topic: 'pipelines',
+  command: 'setup',
+  description: 'bootstrap a new pipeline with common settings',
+  help: `Example:
+
+  heroku pipelines:setup example githuborg/reponame -o example-org
+  ? Automatically deploy the master branch to staging? Yes
+  ? Wait for CI to pass before deploying the master branch to staging? Yes
+  ? Enable review apps? Yes
+  ? Automatically create review apps for every PR? Yes
+  ? Automatically destroy idle review apps after 5 days? Yes
+  Creating pipeline... done
+  Linking to repo... done
+  Creating ⬢ example (production app)... done
+  Creating ⬢ example-staging (staging app)... done
+  Configuring pipeline... done
+  View your new pipeline by running \`heroku pipelines:open e5a55ffa-de3f-11e6-a245-3c15c2e6bc1e\``,
+  needsApp: false,
+  needsAuth: true,
+  args: [
+    {
+      name: 'name',
+      description: 'name of pipeline',
+      optional: true
+    },
+    {
+      name: 'repo',
+      description: 'a GitHub repository to connect the pipeline to',
+      optional: true
+    }
+  ],
+  flags: [
+    {
+      name: 'organization',
+      char: 'o',
+      description: 'the organization which will own the apps (aliased as --team)',
+      hasValue: true
+    },
+    {
+      name: 'team',
+      char: 't',
+      description: 'the team which will own the apps (aliased as --organization)',
+      hasValue: true
+    }
+  ],
+  run: cli.command(co.wrap(function*(context, heroku) {
+    const herokuToken = heroku.options.token
+    const githubToken = yield getGitHubToken(herokuToken)
+    const organization = context.flags.organization || context.flags.team
+
+    const { name: pipelineName, repo: repoName } = yield getNameAndRepo(context.args)
+    const repo = yield getRepo(githubToken, repoName)
+    const settings = yield getSettings(repo.default_branch)
+
+    let ciSettings
+    if (yield hasCIFlag(heroku)) {
+      ciSettings = yield getCISettings(organization)
+    }
+
+    const pipeline = yield cli.action(
+      'Creating pipeline',
+      api.createPipeline(heroku, pipelineName)
+    )
+
+    yield cli.action(
+      'Linking to repo',
+      kolkrabbi.createPipelineRepository(herokuToken, pipeline.id, repo.id)
+    )
+
+    const archiveURL = yield github.getArchiveURL(githubToken, repoName, repo.default_branch)
```

Comment:
We could either just tackle it, if / when it comes up. Or request another archive link after creating the production app.